HypothesisPlugin.inc.php

```diff
@@ -43,6 +44,33 @@ function callback($hookName, $args) {
 		return false;
 	}
 
+	/**
+	 * Hook callback function for TemplateManager::display
+	 * @param $hookName string
+	 * @param $args array
+	 * @return boolean
+	 */
+	function callbackTemplateDisplay($hookName, $args) {
+		if ($hookName != 'TemplateManager::display') return false;
+		$templateMgr = $args[0];
+		$template = $args[1];
+		if ($template == 'plugins-plugins-generic-pdfJsViewer-generic-pdfJsViewer:articleGalley.tpl') {
```

Comment:
Wow, is this really the template name you're observing? That seems wrong...

Reply:
Yes, though it came from an existing 3.1.2 development install. Let me try a vanilla install to confirm, though probably not until tomorrow.

Reply:
I've tested this with 3.2.0 and it does require a small change -- `articleGalley.tpl` became `submissionGalley.tpl`. Otherwise this works. Please tweak that and I'll merge this into `master`!